src/main/java/org/synyx/urlaubsverwaltung/absence/OvertimeAbsenceApiController.java

```diff
@@ -81,7 +82,10 @@ public OvertimeAbsenceDto overtimeAbsence(
             throw new ResponseStatusException(INTERNAL_SERVER_ERROR, "Absence is of category OVERTIME, but duration is not know.");
         }
 
-        return new OvertimeAbsenceDto(absenceId, application.getHours());
+        final var partitionedDurations = applicationService.partitionOvertimeReduction(application);
```

Comment:
Warum lagern wir dies in den Service aus? Das hat ja nur mit Informationen aus der Application zu tun. Würde die ganze Berechnung in die Application packen.